web/docs/api_v1.yaml

```diff
@@ -0,0 +1,126 @@
+openapi: "3.0.2"
+info:
+  title: "NordicValidator"
+  description: "A docker image to validate epub and dtbook"
+  version: '1.0'
+paths:
+  /v1/Validation/:
+    post:
+      summary: "Validates a single file and returns the report as part of the response."
+      description: ""
+      requestBody:
+        required: true
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                config:
+                  $ref: "#/components/schemas/Config"
+                file:
+                  type: string
+                  format: binary
+          application/json:
+            schema:
+              type: object
+              properties:
+                config:
+                  $ref: "#/components/schemas/Config"
+                downloadFilePath:
+                  type: string
+                  description: "Path to epub file stored on OneDrive."
```

Comment:
@kalaspuffar is this OneDrive-specific? If we could reference any absolute path here it would be convenient. We could for instance mount our EPUB archive as a file system in the container, and refer to files there.